attest/net/Cargo.toml

```diff
@@ -26,7 +26,7 @@ displaydoc = "0.2"
 mbedtls = "0.8.1"
 pem = "1.0"
 percent-encoding = "2.1.0"
-reqwest = { version = "0.10" , default-features = false, features = ["rustls-tls", "gzip"] }
+reqwest = { version = "0.10" , default-features = false, features = ["blocking", "rustls-tls", "gzip"] }
```

Comment:
I think this doesn't do anything unless you also pick the blocking module at the call sites

Reply:
Right, we use the `reqwest::blocking::Client` without the corresponding feature.

Reply:
oh i see thanks